client/src/app/my-product/_components/seller-product-card-desktop.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from 'next/image'
+
+import { Text } from '@/components/text'
+
+import type { ArtToy } from '@/generated/api'
+
+export interface SellerProductCardDesktopProps {
+  product: ArtToy
+}
+
+const SellerProductCardDesktop = ({
+  product,
+}: SellerProductCardDesktopProps) => {
+  const date = new Date(product.release_date)
+  const formattedDate = `${date.getDate()}-${date.getMonth()}-${date.getFullYear() + 543}`
```

Comment:
ใช้ useMemo ด้วย ไม่งั้นเราจะ rerender ตลอดเวลาที่ user interact อะไรซักอย่างใน page ทั้งที่ข้อมูลตรงนี้มันเหมือนเดิมเดิม

อาจจะประมาณนี้

```suggestion
  const formattedDate = useMemo(()=>{ 
  const date = new Date(product.release_date)
  return `${date.getDate()}-${date.getMonth()}-${date.getFullYear() + 543}` }, [date])
```